stats/influxdb/util.go

```diff
@@ -57,3 +59,28 @@ func MakeBatchConfig(conf Config) client.BatchPointsConfig {
 		WriteConsistency: conf.Consistency.String,
 	}
 }
+
+func CheckDuplicatedTypeDefinitions(fieldKinds map[string]fieldKind, tag string) {
+	if _, found := fieldKinds[tag]; found {
+		logrus.WithFields(logrus.Fields{
+			"tagName": tag,
+		}).Warn("A tag name shows up more than once in InfluxDB field type configurations.")
+	}
```

Comment:
We are trying to move away from using the global `logrus` and this seems more like an error. 
I am somewhat split on whether we should just warn or error out and abort the script, but the second seems more consistent IMO.
So I would argue this should return an error and that errors should be propagated by `MakeFieldKinds` and then `New`.

Reply:
I see.  I will update the code in that way.  Thanks.